easybuild/easyconfigs/r/ReFrame/ReFrame-3.0.eb

```
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.0'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = False
+
+exts_list = [
+    ('setuptools', '47.1.1', {
+        'source_tmpl': 'setuptools-47.1.1.zip',
```

@teojgo Please use `%(version)s` here to avoid hardcoding the `setuptools` version in two places:

```suggestion
        'source_tmpl': 'setuptools-%(version)s.zip',
```